virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -63,10 +65,68 @@ def metabolic_rate(
         raise ValueError("Invalid metabolic type: {metabolic_type}")
 
 
+def metabolic_rate(
+    mass: float,
+    temperature: float,
+    terms: dict,
+    metabolic_type: MetabolicType,
+) -> float:
+    """Calculates metabolic rate in grams of body mass per day.
+
+    This follows the Madingley implementation, assuming a power-law relationship with
+    mass and an exponential relationship with temperature.
+
+    TODO: Implement activity windows to properly paramterize sigma.
+    TODO: Move constants to constants file after constants rework.
+
+    Args:
+        mass: The body-mass [kg] of an AnimalCohort.
+        temperature: The temperature [Celsius] of the environment.
+        terms: The tuple of metabolic rate terms used.
+        metabolic_type: The metabolic type of the animal [ENDOTHERMIC or ECTOTHERMIC].
+
+    Returns:
+        The metabolic rate of an individual of the given cohort in [g/d].
+    """
+
+    Es = 3.7 * 10 ** (-2)  # energy to mass conversion constant (g/kJ)
+    sig = 0.5  # proportion of time-step with temp in active range (toy)
+    Ea = 0.69  # aggregate activation energy of metabolic reactions
```

Comment:
Does this conform with the decided approach of including constants? If not, the standardised approach should be used.

Reply:
It does not! I am doing a full constants rework immediately after this PR to avoid the PR sprawling out into the whole animal module.